app/src/test/java/edu/nextstep/camp/calculator/domain/ExpressionTest.kt

```diff
@@ -7,7 +7,7 @@ class ExpressionTest {
     @Test
     fun `빈 수식일 때, 피연산자를 추가할 수 있어야한다`() {
         // given
-        val expression = Expression.EMPTY
+        val expression = edu.nextstep.camp.calculator.domain.Expression.EMPTY
```

Comment:
```suggestion
        val expression = Expression.EMPTY
```
import를 하시게 되면 풀 패키지명을 붙이지 않고 사용이 가능합니다. (이하 관련 내용 동일)